CoC.md

```diff
@@ -114,6 +114,77 @@ following behaviors:
   any other violation of this code. That includes trolling and other forms of
   consistently disruptive behavior.
 
+## Enforcement
+
+We do not tolerate unacceptable behavior from any community member, and there
+are no exceptions for those in positions of power such as maintainers,
+sponsors, funders, or other individuals withe decision making authority.
+Further, people in positions of power can wield it to exacerbate the effects of
+harassment and to diminish the repercussions, those who are informal or formal
```

Comment:
I would suggest: "and to diminish the repercussions. For these reasons, those who are informal .."